includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -2045,15 +2049,29 @@ private function prepare_payment_information_from_request( WC_Order $order ) {
 			$payment_information['statement_descriptor_suffix'] = WC_Stripe_Helper::get_dynamic_statement_descriptor_suffix( $order );
 		}
 
+		$payment_method_options = [];
+
 		// Specify the client in payment_method_options (currently, Checkout only supports a client value of "web")
 		if ( 'wechat_pay' === $selected_payment_type ) {
-			$payment_information['payment_method_options'] = [
+			$payment_method_options = [
 				'wechat_pay' => [
 					'client' => 'web',
 				],
 			];
 		}
 
+		// Add the updated preferred credit card brand when defined
+		$preferred_brand = $payment_information['payment_method_details']->card->networks->preferred ?? null;
```

Comment:
We could use `$payment_method_object` instead of `$payment_information['payment_method_details']`, which got me thinking that maybe `$payment_method_details` is a more descriptive name? 🤷🏼 
```suggestion
		$preferred_brand = $payment_method_object->card->networks->preferred ?? null;
```